lib/common/utils/document.js

```diff
@@ -118,22 +121,149 @@ define(['jquery', 'async', 'underscore'], function ($, async, _) {
             return this;
         },
 
+        addPageTag: function (ctx, isServer, name, attributes, content) {
+
+            var pageTag = this._createPageTag(name, attributes, content);
+
+            if (isServer) {
```

Comment:
@adam-26 What is the reason for storing the page tags in `ctx._rootCtx.pageTags` on the server, but `ctx.meta.pageTags` on the client?